lesson_03/Homework_03_Dalichuk.py

```diff
@@ -0,0 +1,121 @@
+# task 01 == Розділіть змінну alice_in_wonderland так, щоб вона займала декілька фізичних лінії
+# task 02 == Знайдіть та екрануйте всі символи одинарної дужки у тексті
+# task 03 == Виведіть змінну alice_in_wonderland на друк
+alice_in_wonderland = '"Would you tell me, please, which way I ought to go from here?"\n"That depends a good deal on where you want to get to," said the Cat.\n"I don\'t much care where ——" said Alice.\n"Then it doesn\'t matter which way you go," said the Cat.\n"—— so long as I get somewhere," Alice added as an explanation.\n"Oh, you\'re sure to do that," said the Cat, "if you only walk long enough."'
+print(alice_in_wonderland)
+
+"""
+    # Задачі 04 -10:
+    # Переведіть задачі з книги "Математика, 5 клас"
+    # на мову пітон і виведіть відповідь, так, щоб було
+    # зрозуміло дитині, що навчається в п'ятому класі
+"""
+# task 04
+"""
+Площа Чорного моря становить 436 402 км2, а площа Азовського
+моря становить 37 800 км2. Яку площу займають Чорне та Азов-
+ське моря разом?
+"""
+Black_Sea_S: int = 436402
+Azov_Sea_S: int = 37800
+Both_Seas_S: int = Black_Sea_S + Azov_Sea_S
```

Comment:
```suggestion
black_sea_s: int = 436402
azov_sea_s: int = 37800
both_seas_s: int = black_sea_s + azov_sea_s
```
змінні завжди з маленької і маленькими

Reply:
дякую дуже! не буду перезасилати комміт. зрозуміла.